SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+To report a vulnerability or a security issue please fill the security advisories form
```

Comment:
It would be great if we can stick to 80 char line limits in our markdown files, if possible :)